connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1209,11 +1214,11 @@ ble_error_t Gap::reset()
     /* Notify that the instance is about to shut down */
 //    shutdownCallChain.call(this);
     shutdownCallChain.clear();
+    _event_queue.clear();
```

Comment:
This event queue can be shared with other components, that would clear the events they have registered.

Reply:
It's not used outside of gap. If you want me to remove this then I have to deprecate all individual resets and this will take time/

Reply:
Can we return the event id in the post and have a cancel method ?